src/firefly/js/ui/VersionInfo.jsx

```diff
@@ -82,26 +91,27 @@ function VersionInfoFull() {
                 </div>
             }
             <Acknowledgement/>
-        </div>
+        </Stack>
     );
 }
 
 const Acknowledgement= () => (
-    <div style={{padding:'10px 5px 3px 5px', width:520, fontSize:'85%', lineHeight:1.3}}>
+
+    <Typography level='body-xs' width={'100%'} sx={{pt:0.6,pr:0.3, pb:0.2, pl:0.3, lineHeight:1.3}}>
             Firefly development by&nbsp;
-        <a href='https://ipac.caltech.edu' target='ipac-window'>IPAC</a>
+        <Link underline='always' color='primary' href='https://ipac.caltech.edu' target='ipac-window'>IPAC</Link>
             &nbsp;at&nbsp;
-        <a href='https://www.caltech.edu' target='caltech-window'>Caltech</a>
+        <Link underline='always' color='primary' href='https://www.caltech.edu' target='caltech-window'>Caltech</Link>
             &nbsp;has been supported by NASA, principally through&nbsp;
-        <a href='https://irsa.ipac.caltech.edu' target='ipac-window'>IRSA</a>
+        <Link underline='always' color='primary' href='https://irsa.ipac.caltech.edu' target='ipac-window'>IRSA</Link>
            , and by the National Science Foundation, through the&nbsp;
-        <a href='https://www.lsst.org/' target='rubin-window'>Vera C. Rubin Observatory</a>
+        <Link underline='always' color='primary' href='https://www.lsst.org/' target='rubin-window'>Vera C. Rubin Observatory</Link>
             . Firefly is open-source software, available on&nbsp;
-        <a href='https://github.com/Caltech-IPAC/firefly' target='github-window'>GitHub</a>
+        <Link underline='always' color='primary' href='https://github.com/Caltech-IPAC/firefly' target='github-window'>GitHub</Link>
         &nbsp;and&nbsp;
-        <a href='https://hub.docker.com/repository/docker/ipac/firefly' target='dockerhub-window'>DockerHub</a>
+        <Link underline='always' color='primary' href='https://hub.docker.com/repository/docker/ipac/firefly' target='dockerhub-window'>DockerHub</Link>
```

Comment:
Modify `ThemeSetup.js` to setup the default link. then take out the properties
from `<Link/>`
```js
 JoyLink: {
     defaultProps: {
        underline: 'always',
        color: 'primary'
  }
```